### DISCUSSION TITLE
CMI customrecipe compatibility problem and suggestion

### DISCUSSION THREAD

**Discussion by Fargus-Van-Crystal-de-Mindark** [Zrips/CMI#5069] [Feature Requests]
Hello, hope to find you well, I have a compatibility problem in game... And I would like to describe to you the situation and suggest a solution that could help a lot of players... I hope that you understand what I will describe, because I'm not good in english.

Ok thanks for listening... :)

in CMI there is a feature /cmi customrecipe that let you create new recipes. But it is only minecraft edition... I would like to use IA items and slimefun too to create new crafting recipes... Do you think that this will be possible?
Itemsadder has a free open api they can use without even owning the plugin at, all it's available here https://github.com/LoneDev6/API-ItemsAdder

https://itemsadder.plugin.ga/developers/java-api/examples
**This explains how to get itemsadder items easily with 1 line code**

Could you try to look at this? This will be wonderfool if you can let players use this feature without installing other plugins that do the exactly the same if we can already use CMI.

**Comment by CloudeLecaw** [comment]
u know request here got ignored by the devs mostly? This here is jsut for issues
send them here aggain: https://github.com/Zrips/CMI/discussions

**Reply by Fargus-Van-Crystal-de-Mindark** [reply]
this is ignored also in discussion my friend :)

**Comment by Fargus-Van-Crystal-de-Mindark** [comment]
https://github.com/Zrips/CMI/discussions/5031
